applicationset/controllers/applicationset_controller.go

```diff
@@ -1508,12 +1510,12 @@ func getOwnsHandlerPredicates(enableProgressiveSyncs bool) predicate.Funcs {
 		},
 		GenericFunc: func(e event.GenericEvent) bool {
 			if log.IsLevelEnabled(log.DebugLevel) {
-				var appName string
+				logFields := log.Fields{"app": ""}
```

Comment:
```suggestion
				logFields := log.Fields{}
```

Probably fine to start with an empty set and replace it if the type assertion is successful.